public/diagnostic.js

```diff
@@ -1,5 +1,56 @@
 /// Ember Object Diagnostic ///
 
+const Order = Ember.Object.extend({
+  price: undefined,
```

Comment:
Close. You need to set ember objects to refer to themselves, which allows ember to create getters and setters for them. 
```js
price: this.price,
qty: this.qty,
```